python/rmm/_lib/cuda_stream_view.pxd

```diff
@@ -20,8 +20,13 @@ cdef extern from "rmm/cuda_stream_view.hpp" namespace "rmm" nogil:
     cdef cppclass cuda_stream_view:
         cuda_stream_view()
         cuda_stream_view(cudaStream_t)
+        cudaStream_t value()
         bool is_default()
         bool is_per_thread_default()
         void synchronize() except +
 
     cdef bool operator==(cuda_stream_view const, cuda_stream_view const)
+
+    const cuda_stream_view cuda_stream_default
```

Comment:
So weird that these don't have to be declared as functions!

Reply:
Why would they be? Aren't these constexpr variables in C++?

Reply:
I'm sorry, I was thinking they were `constexpr` functions in C++. But they are variables, not functions. But note only one of them is `constexpr` (because you can't have a C-style cast (`reinterpret_cast`) in a `constexpr` expression, and CUDA defines `cudaStreamPerThread` using a c-style cast.